doc/getting_started/build_sw.md

```diff
@@ -121,9 +121,9 @@ You may find it useful to use wildcards to build/test all targets in the OpenTit
 If a a target (a test or build artifact) relies on optional parts of the "Getting Started" guide they should be tagged so they can be filtered out and users can `bazelisk.sh test //...` once they filter out the appropriate tags.
 We maintain or use the following tags to support this:
 * `broken` is used to tag tests that are committed but should not be expected by CI or others to pass.
-* `cw310_test_rom`, `cw310_rom`, and `cw310_other` are used to tag tests that depend on a correctly setup cw310 "Bergen Board" to emulate OpenTitan.
-  The `cw310` prefix may be used in `--test_tag_filters` to enable concise filtering.
-  Loading the bitstream is the slowest part of the test, so these tags can group tests with common bitstreams to accelerate the tests.
+* `cw310`, `cw310_test_rom`, and `cw310_rom` are used to tag tests that depend on a correctly setup cw310 "Bergen Board" to emulate OpenTitan.
+  The `cw310` tag may be used in `--test_tag_filters` to enable concise filtering to select tests that run on this board and include or exclude them.
```

Comment:
In this context, `cw310` is actually interpreted as a tag name, not a regex, right?

Reply:
yes, I was originally confused because of the regex buisiness, but the cw310 tag is also in use and applied generally and the *_tag_filters appear to work on the tags (unlike queries).